### ISSUE TITLE
ruby-v2_2_3-src has a different hash every time I try to download it

### ISSUE THREAD

**Issue by peti** [NixOS/nixpkgs#11542]
It appears that the downloaded tarball has a different SHA hash every time. I tried to fix `pkgs/development/interpreters/ruby/default.nix` but had no success:

~~~~~~~
$ nix-build --no-out-link ~/.nix-defexpr -A ruby_2_2_3
these derivations will be built:
  /nix/store/lkgqbdjinjpp09rp4cwi9wmnc6i1x2sa-ruby-v2_2_3-src.drv
  /nix/store/zbwf9jhgzxf2jkj0nivrqz4q6biib8y5-ruby-2.2.3-p0.drv
these paths will be fetched (0.00 MiB download, 39.33 MiB unpacked):
  /nix/store/4z87f91jpy12fnksc1c4g1bpa2ydgczk-ruby-2.2.3-p0
fetching path ‘/nix/store/4z87f91jpy12fnksc1c4g1bpa2ydgczk-ruby-2.2.3-p0’...
building path(s) ‘/nix/store/1m4hbbd2a7j7gm460416csa0ibvxv0v6-ruby-v2_2_3-src’

*** Downloading ‘http://hydra.cryp.to/nar/4z87f91jpy12fnksc1c4g1bpa2ydgczk-ruby-2.2.3-p0’ (signed by ‘hydra.cryp.to-1’) to ‘/nix/store/4z87f91jpy12fnksc1c4g1bpa2ydgczk-ruby-2.2.3-p0’...
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
trying https://github.com/ruby/ruby/archive/v2_2_3.tar.gz
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100   117    0   117    0     0    209      0 --:--:-- --:--:-- --:--:--   219
100 11.9M  100 11.9M    0     0  1679k      0  0:00:07  0:00:07 --:--:-- 2409k
unpacking source archive /tmp/nix-build-ruby-v2_2_3-src.drv-0/v2_2_3.tar.gz
100 2605k    0 2605k    0     0   336k      0 --:--:--  0:00:07 --:--:--  505k
output path ‘/nix/store/1m4hbbd2a7j7gm460416csa0ibvxv0v6-ruby-v2_2_3-src’ should have r:sha256 hash ‘126b2diaa9zslb1yynr6351vy32wvwjwjs6ai2pcj8zg675028x6’, instead has ‘1ssq3c23ay57ypfis47y2n817hfmb71w0xrdzp57j6bv12jqmgrx’
killing process 4794
~~~~~~~

**Comment by copumpkin** [comment]
Is this fixed by 6945557dba4525c41bf3e6b04e0d588a4bbd8432?

**Comment by peti** [comment]
I think it is. Thanks!
